tests/test_ensdf_processing.py

```diff
@@ -163,8 +226,8 @@ def file_comp(file_out, file_ref, exceptions):
         type 4: carriage return vs. non standard return type.
             options: line number of return.
     '''
-    f_out = open(file_out, 'r')
-    f_ref = open(file_ref, 'r')
+    f_out = open(file_out, 'r',encoding="latin-1")
+    f_ref = open(file_ref, 'r',encoding="latin-1")
```

Comment:
It is not clear why you need this encoding.  Wouldn't utf8 or ascii be more applicable?